eq-author/src/App/history/HistoryPage.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styled from "styled-components";
+import { useQuery } from "@apollo/react-hooks";
+
+import CustomPropTypes from "custom-prop-types";
+
+import Error from "components/Error";
+import Loading from "components/Loading";
+
+import HistoryItem from "./HistoryItem";
+import Header from "components/EditorLayout/Header";
+import { Grid } from "components/Grid";
+import questionnaireHistoryQuery from "./questionnaireHistory.graphql";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+`;
+
+const StyledGrid = styled(Grid)`
+  display: flex;
+  flex-direction: column;
+  padding: 1em;
+  border: 1px solid #d6d8da;
+  margin: 0.5em 1em;
+  overflow: hidden;
+  max-width: 60em;
+`;
+
+export const UnwrappedHistoryPageContent = ({ match }) => {
+  const { loading, error, data } = useQuery(questionnaireHistoryQuery, {
+    variables: { input: { questionnaireId: match.params.questionnaireId } },
+  });
+
+  let createdAt, userName, questionnaireTitle;
+
+  if (loading) {
+    return <Loading height="100%">Questionnaire history loading…</Loading>;
+  }
+
+  if (error) {
+    return <Error>Oops! Something went wrong</Error>;
+  }
+
+  if (data && !loading) {
```

Comment:
Don't think we need this if statement here, if loading and errors are false and data doesn't  we want it to throw as it means something is very wrong somewhere.

Also if we take it out theres no need for the decelerations on 36